src/core/BufferAttribute.js

```diff
@@ -37,7 +39,7 @@ BufferAttribute.prototype = {
 
 	get count() {
 
-		return this.array.length / this.itemSize;
+		return ( this.discardedLength > 0 ? this.discardedLength : this.array.length ) / this.itemSize;
```

Comment:
true, actually the original calculation is pointless unless I am missing something subtle, the Typed Array is fixed length, so we know the item count at the time of construction. Calculate once there as itemCount and use for that value in the getter. Preserving the getter to prevent accidental changing I assume?

The same applies to InterleavedBufferAttributes count as far as I can see. Not having used them myself.